public/app/plugins/datasource/cloudwatch/datasourcecfg.gen.ts

```diff
@@ -0,0 +1,42 @@
+// Code generated - EDITING IS FUTILE. DO NOT EDIT.
+//
+// Generated by:
+//     public/app/plugins/gen.go
+// Using jennies:
+//     TSTypesJenny
+//     PluginTSTypesJenny
+//
+// Run 'make gen-cue' from repository root to regenerate.
+
+import * as common from '@grafana/schema';
+
+import * as common from '@grafana/schema';
```

Comment:
Issue: duplicated import.

This shows up when `common` is imported in `datasourcecfg.cue` like the following

```cue
import (
	common "github.com/grafana/grafana/packages/grafana-schema/src/common"
	"github.com/grafana/grafana/pkg/plugins/pfs"
)
```